src/test/utils.ts

```diff
@@ -7,3 +8,20 @@ export const mockedTAccessToken: TAccessToken = {
   iss: "urn:grindery:orchestrator",
   exp: 1690552568,
 };
+
+export const mockedConnectorInput: ConnectorInput<any> = {
+  sessionId: "mySessionId",
+  cdsName: "myCdsName",
+  key: "totalSupplyAction",
+  fields: {
+    chain: "eip155:5",
+    contractAddress: "0xD6dAC59F68089CE0c82310Ec213Ac9E25561a5f0",
+    parameters: {
+      to: "0x388C818CA8B9251b393131C08a736A67ccB19297",
+      value: "1000",
+    },
+    functionDeclaration: "function transfer(address to, uint256 value) public virtual returns (bool) ",
+    userToken:
+      "eyJhbGciOiJFUzI1NiJ9.eyJhdWQiOiJ1cm46Z3JpbmRlcnk6YWNjZXNzLXRva2VuOnYxIiwic3ViIjoiZWlwMTU1OjE6MHgxMEEyQzMwNmNDYzg3OTM4QjFmZTNjNjNEQmIxNDU3QTljODEwZGY1IiwiaWF0IjoxNjg3MjcwOTk1LCJpc3MiOiJ1cm46Z3JpbmRlcnk6b3JjaGVzdHJhdG9yIiwiZXhwIjoxNjg3Mjc0NTk1fQ.WUEC1GFkRACK7rdwcV0kt08_m4-YDzifkWPWcdhsVzDAunevAnDD5mqILDX7Czn92eMUZy1hb3IFPJQyTzNQnw",
```

Comment:
I would suggest to use `TAccessToken.sign` to generate a token instead of hardcoding it. Note that TAccesToken needs to be refactored to be constructed by `typedToken`.

Reply:
Actually, no refactor needed, should be `AccessToken.sign`.

Reply:
@SAPikachu Done, I've used the `signJWT` method that is provided by `jwtTools`. Seems good to me as this is non-critical because in the unit tests, we will never seek to know the validity of the token, it could even be expired, we will always mock this part.